src/NuGet.Core/NuGet.Build.Tasks.Pack/NuGet.Build.Tasks.Pack.targets

```diff
@@ -169,19 +169,12 @@ Copyright (c) .NET Foundation. All rights reserved.
       <NuGetPackInput Include="@(_TargetPathsToSymbols->'%(FinalOutputPath)')"/>
       <NuGetPackInput Include="@(_SourceFiles)"/>
       <NuGetPackInput Include="@(_References)"/>
-      <NuGetPackOutput Include="$(RestoreOutputAbsolutePath)$(PackageId).$(PackageVersion).nuspec"/>
-
-      <NuGetPackOutput Include="$(PackageOutputAbsolutePath)$(PackageId).$(PackageVersion).nupkg"
-                       Condition="'$(ContinuePackingAfterGeneratingNuspec)' == 'true'"/>
-
-      <NuGetPackOutput Include="$(PackageOutputAbsolutePath)$(PackageId).$(PackageVersion).symbols.nupkg"
-                       Condition="'$(IncludeSource)' == 'true'
-                               OR '$(IncludeSymbols)' == 'true'"/>
-
-      <NuGetPackOutput Include="$(RestoreOutputAbsolutePath)$(PackageId).$(PackageVersion).symbols.nuspec"
-                       Condition="'$(IncludeSource)' == 'true'
-                             OR '$(IncludeSymbols)' == 'true'"/>
     </ItemGroup>
+
+    <MSBuild Projects="$(MSBuildProjectFullPath)"
```

Comment:
@jp2masa sorry for getting back to you on this so late (i was on vacation). we avoid invoking msbuild in targets where it can be avoided as it might break incremental builds.

what i would suggest is to make the target _CalculateInputsOutputsForPack depend on the target _GetOutputItemsFromPack which will make @(_OutputPackItems) available to you in the _CalculateInputsOutputsForPack,

You can then just write NuGetPackOutput Include="@(_OutputPackItems)